### PR TITLE
2.5.0 Release

### PR DESCRIPTION
**Pull request by nblumhardt**
* #939, #946 - RTM .NET Standard/Core tooling
 * #955 - `LoggerSinkConfiguration.Wrap()`
 * #944 - `{Properties}` in output templates
 * #961 - fix parser exeception when property tokens with zero-length names are encountered
 * #773 - `LogContext.Clone()` and `LogContext.Push(ILogEventEnricher)`
 * #976 - support for logging C# 7 `ValueTuple<>` values
 * #977 - output formatting performance improvements, `{Message:l}` (unquoted string) and `{Message:j}` (JSON structure) support in output templates
 * #981 - reduced allocations required for structured data capturing

Plus multiple build/test/configuration improvements.

### CODE REVIEW

File: src/Serilog/Serilog.csproj [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <PackageReference Include="System.Threading" Version="4.0.11" />
+  </ItemGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net45' ">
+    <DefineConstants>$(DefineConstants);REMOTING;HASHTABLE</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net46' ">
+    <DefineConstants>$(DefineConstants);ASYNCLOCAL</DefineConstants>
+  </PropertyGroup>
```
Comment (skomis-mm) [inline src/Serilog/Serilog.csproj]: `dotnet migrate` issue: `hashtable` constant missing for this target

Comment (nblumhardt) [inline src/Serilog/Serilog.csproj]: 👍 just pushed a fix

**Comment by nblumhardt** [comment]
I think it's time to push this out. Last-minute thoughts?

**Comment by merbla** [comment]
Go for it

**Comment by nblumhardt** [comment]
Oops, missed #972 in the notes; added.

**Pull request merged**